games-misc/an-anime-game-launcher-bin/an-anime-game-launcher-bin-2.3.8-r2.ebuild

```diff
@@ -39,25 +38,48 @@ src_unpack() {
 src_prepare(){
 	chmod +x ${P}.AppImage
 	./${P}.AppImage --appimage-extract || die "Extraction Failed"
-	chrpath -d "squashfs-root/public/discord-rpc/discord-rpc" || die "Patching Library Failed"
 	default
+
 	mv "squashfs-root/public/icons/256x256.png" "${PN}.png" || die
 	mv "squashfs-root/AppRun" "${PN}" || die
 	mv "squashfs-root/an-anime-game-launcher.desktop" "${PN}.desktop" || die
 }
 
 src_install(){
+
 	insinto "/usr/lib/${PN}"
 	doins "squashfs-root/resources.neu"
 	exeinto "/usr/lib/${PN}"
 	doexe "squashfs-root/an-anime-game-launcher"
-	doins -r "squashfs-root/public"
+
+	insinto "/usr/lib/${PN}/public" # specify path for public folders
+
+	#copy all files/folders from squashfs-root/public
+	doins -r "squashfs-root/public/discord-rpc"
+	doins -r "squashfs-root/public/dxvks.yaml"
+	doins -r "squashfs-root/public/hdiffpatch"
+	doins -r "squashfs-root/public/icons"
+	doins -r "squashfs-root/public/images"
+	doins -r "squashfs-root/public/locales"
+	doins -r "squashfs-root/public/neutralino.js"
+	doins -r "squashfs-root/public/runners.yaml"
+	doins -r "squashfs-root/public/shaders"
+
+	#create pix
 	insinto "/usr/share/pixmaps"
 	doins "${PN}.png"
+	#create bin
 	exeinto "/usr/bin"
 	doexe "${PN}"
+	#create desktop entry
 	insinto "/usr/share/applications/"
 	doins "${PN}.desktop"
+
+	cd ${FILESDIR}
+	insinto "/usr/lib/${PN}/public/discord-rpc/" # set path for discord-rpc binary
```

Comment:
What do we do on line 58 then?

Reply:
We copy the rpc-folder sourced from the appimage first, then we replace it with the one in FILESDIR folder.

Reply:
Oh yes, my bad. 
Sorry about that, I thought line 58 installs the actual binary, since it has the same name like the folder I was mistaken.